src/index.html

```diff
@@ -11,7 +11,187 @@
+            type="checkbox"
+            id="TermOfTheAgreement"
+          >
+        </div>
+      </fieldset>
+      <fieldset class="form-fieldset-second">
+        <legend>Registration</legend> 
+        <div class="form-field">
+          <label for="email">
+            E-mail:
+          </label>
+          <input
+            name="E-mail"
+            type="email" 
+            id="email"
+            placeholder="email@example.com"
+          >
+        </div>
+        <div class="form-field-last-in-fieldset">
+          <label for="password">
+            Password:
+          </label>
+          <input
+            name="Password"
+            type="password"
+            id="password"
+          >
+        </div> 
+      </fieldset>
+      <fieldset class="form-fieldset-third">
+        <legend>An interesting fact about you!</legend> 
+        <div class="form-field">
+          Do you love cats?
+          <label>
+            <input
+              name="LoveCats"
+              type="radio"
+              value="yes"
+            >
+            Yes
+          </label>
+          <label>
+            <input
+              name="LoveCats"
+              type="radio"
+              value="no"
+            >
+            No
+          </label>
+        </div>
+        <div class="form-field">
+          <label for="FavoriteColor">
+            What is your favorite color?
+          </label>
+          <input
+            name="FavoriteColor"
+            type="color"
+            id="FavoriteColor"
+          >
+        </div>
+        <div class="form-field">
+          <label for="GoToBed">
+            What time do you go to bed?
+          </label>
+          <input
+            name="GoToBed"
+            type="time"
+            id="GoToBed"
+          >
+        </div>
+        <div class="form-field">
+          <label>
+            What are your favorite brand of cars?
+            <select
+              name="FavoriteBrandOfCars"
+              required
+              multiple
+            >
+              <option value="BMW">BMW</option>
+              <option value="Audi">Audi</option>
+              <option value="Lada">Lada</option>
+            </select>
+          </label>
+        </div>
+        <div class="form-field-last-in-fieldset">
+          <label for="RateOurWork">
+            How do you rate our work?
+          </label>
+          <input
+            name="RateOurWork" 
+            type="range"
+            id="RateOurWork"
+            min="0"
+            max="100"
+            step="5"
+          >
+        </div> 
+      </fieldset> 
+      <fieldset class="form-fieldset-fourth">
+        <legend>Additional info:</legend> 
+        <div class="form-field">
+          <label for="Comments">
+            Comments:
+          </label>
+          <textarea
+            minlength="0"
+            maxlength="100"
+            name="Comments"
+            id="Comments"
+          >
+          </textarea>
```

Comment:
```suggestion
          <textarea
            minlength="0"
            maxlength="100"
            name="Comments"
            id="Comments"
          ></textarea>
```
To avoid it 
![image](https://user-images.githubusercontent.com/97547824/204133078-de9ed840-e8d9-4c32-afc3-dc5a10650eb7.png)